src/main/java/Event.java

```diff
@@ -0,0 +1,23 @@
+public class Event extends Task {
+    protected String from;
+    protected String to;
+
+    public Event (String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    @Override
+    public void printTask () {
+        System.out.println (getTypeIcon() + getStatusIcon()+ " " + this.description + " (from: " + this.from + " to: "+ this.to + ")");
```

Comment:
Try to keep line length shorter than 110 characters! (Current character count: 127). If the line exceeds the limit, try using line wrapping at appropriate places of the line :)